test/e2e/utils.go

```diff
@@ -0,0 +1,192 @@
+package e2e
+
+import (
+	operator "github.com/instaclustr/cassandra-operator/pkg/apis/cassandraoperator/v1alpha1"
+	"github.com/instaclustr/cassandra-operator/pkg/sidecar"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"os"
+	"testing"
+	"time"
+)
+
+const (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 500
+	cleanupRetryInterval = time.Second * 3
+	cleanupTimeout       = time.Second * 60
+)
+
+func initialise(t *testing.T) (*framework.TestCtx, *framework.Framework, *framework.CleanupOptions) {
+	ctx := framework.NewTestCtx(t)
+
+	if err := ctx.InitializeClusterResources(&framework.CleanupOptions{TestContext: ctx, Timeout: timeout}); err != nil {
+		t.Fatalf("Failed to initialize cluster resources: %v", err)
+	}
+
+	t.Log("Initialized cluster resources")
+
+	namespace, err := ctx.GetNamespace()
+
+	if err != nil {
+		t.Fatalf("Could not obtain namespace: %v", err)
+	}
+
+	f := framework.Global
+
+	if err := e2eutil.WaitForOperatorDeployment(t, f.KubeClient, namespace, "cassandra-operator", 1, retryInterval, timeout); err != nil {
+		t.Fatal("Timeout of Cassandra operator deployment has occurred.", err)
```

Comment:
What's the diff between `t.Fatalf` and `t.Fatal`? ie line 29 vs line 43?

Reply:
Fatal is equivalent to Log followed by FailNow. Fatalf is equivalent to Logf followed by FailNow. Log is just a logger, with Logf, you can format that message. In this particular case, it should be Fatalf.